### ISSUE TITLE
Recording Audion in the Channel and trying to write to a WAV file.

### ISSUE THREAD

**Issue by TheCynosure** [DV8FromTheWorld/JDA#90]
We are developing a discord bot and we open the an audio connection and use an AudioReceiveHandler to record audio from the channel. Then when writing it to the wav or before we get this error with the JDA Library.

```
[17:32:54] [Fatal] [JDAAudioConn]: Encountered an exception:
java.lang.NegativeArraySizeException
    at net.dv8tion.jda.audio.Decoder.decodeFromOpus(Decoder.java:79)
    at net.dv8tion.jda.audio.AudioConnection$3.run(AudioConnection.java:366)
```

**Comment by Almighty-Alpaca** [comment]
This is clearly an issue on JDA's site. It's caused by the OpusDecoder having an error which we don't handle at all.
As @DV8FromTheWorld made the audio receiving system a few days ago so he is probably the best guy for fixing it.

**Comment by DV8FromTheWorld** [comment]
This is only possible if the audio provided to JDA from Discord cannot properly be decoded by Opus. This error is now handled and a log message is sent at level `TRACE`.
Fixed as of version: `2.2.0_337`

**Comment by TheCynosure** [comment]
This is awesome, thank you!
